factoring/interfaces.py

```diff
@@ -74,7 +92,7 @@ def factor(P, use_saved_embedding=True):
     sample_time = time.time()
 
     # get QPU sampler
-    sampler = DWaveSampler()
+    sampler = DWaveSampler(solver_features=dict(online=True, name='DW_2000Q.*'))
```

Comment:
The current naming for online solvers in prod is DW_2000Q_ suffixed by 2 or 3

Reply:
Yes, but also for example DW_2000Q_VFYC_2, and I don't know that we have pre-embeddings for them

Reply:
That's a very valid point :)

I'm not sure what's the list Leap users will see?

If VFYC is included, we can use `DW_2000Q_[12]` for regex.

Reply:
According to Vlad and Pete, they will see only DW_2000Q_2 and DW_2000Q_3, that is what they just Slacked me now

Reply:
:+1:

That's why I would just get rid of the complete name part in prod. @tcnleung